doc/user-manual/language/universe-levels.lagda.rst

```diff
@@ -197,6 +197,29 @@ Pragmas and options
   Setω`` (thus making Agda inconsistent) but otherwise leaves universe
   checks intact.
 
+.. _level-universe:
+
+* The option :option:`--level-universe` makes ``Level`` live in its own
+  universe ``LevelUniverse`` and disallows having levels depend on terms
+  that are not levels themselves. When this option is turned off,
+  ``LevelUniverse`` still exists, but reduces to ``Set``.
+  Note: While compatible with the `--cubical` option, this option is currently
+  not compatible with cubical builtin files, and an error will be raised when
+  trying to import them in a file using `--level-universe`.
+  .. code-block:: agda
```

Comment:
You will need an empty line before `.. code-block`.  See https://agda--6532.org.readthedocs.build/en/6532/language/universe-levels.html#level-universe